RFC-007-SWAP-Ethereum-basic.md

```diff
@@ -0,0 +1,315 @@
+# Ethereum Basic HTLC Atomic Swap
+
+- RFC-Number: 007
+- Status: Draft
+- Discussion-issue: -
+- Created on: 2019-03-12
+
+## Table of Contents
+<!-- toc -->
+- [Description](#description)
+- [The Ethereum Identity](#the-ethereum-identity)
+- [Hash Time Lock Contract](#hash-time-lock-contract)
+    - [Hash Functions](#hash-functions)
+    - [Parameters](#parameters)
+    - [Contract](#contract)
+- [Execution Phase](#execution-phase)
+    - [Deployment](#deployment)
+    - [Redeem](#redeem)
+    - [Refund](#refund)
+- [Registry extension](#registry-extension)
+- [Examples](#examples)
+    - [RFC003 SWAP REQUEST](#rfc003-swap-request)
+    - [RFC003 SWAP RESPONSE](#rfc003-swap-response)
+    - [HTLC](#htlc)
+
+<!-- tocstop -->
+
+## Description
+
+This RFC defines how to execute a [RFC003](./RFC-003-SWAP-basic.md) SWAP where one of the ledgers is Ethereum and the associated asset is the native Ether asset.
+
+For definitions of the Ethereum ledger and Ether asset see [RFC006](./RFC-004-SWAP-Ethereum.md).
+
+To fulfil the requirements of [RFC003](./RFC-003-SWAP-basic.md) this RFC defines:
+
+- The [identity](./RFC-003-SWAP-basic.md#identity) to be used when negotiating a SWAP on the Ethereum ledger.
+- How to construct a Hash Time Lock Contract (HTLC) to lock the Ether asset on the Ethereum blockchain.
+- How to deploy, redeem and refund the HTLC during the execution phase of [RFC003](./RFC-003-SWAP-basic.md).
+
+## The Ethereum Identity
+
+The Identity to be used on Ethereum is the Ethereum *address* as defined in section 284 of the [Ethereum Yellowpaper](https://ethereum.github.io/yellowpaper/paper.pdf).
```

Comment:
```suggestion
The identity to be used on Ethereum is the Ethereum *address* as defined in section 284 of the [Ethereum Yellowpaper](https://ethereum.github.io/yellowpaper/paper.pdf).
```

Let's be consistent on the casing. I don't mind either way (capitalised or not).